packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/loadTestRun/api/operations.ts

```diff
@@ -154,242 +156,22 @@ export async function _testRunDeserialize(
-                  : {
-                      url: result.body.testArtifacts?.outputArtifacts
-                        ?.resultFileInfo?.["url"],
-                      fileName:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.resultFileInfo?.["fileName"],
-                      fileType:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.resultFileInfo?.["fileType"],
-                      expireDateTime:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.resultFileInfo?.["expireDateTime"],
-                      validationStatus:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.resultFileInfo?.["validationStatus"],
-                      validationFailureDetails:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.resultFileInfo?.["validationFailureDetails"],
-                    },
-                logsFileInfo: !result.body.testArtifacts?.outputArtifacts
-                  ?.logsFileInfo
-                  ? undefined
-                  : {
-                      url: result.body.testArtifacts?.outputArtifacts
-                        ?.logsFileInfo?.["url"],
-                      fileName:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.logsFileInfo?.["fileName"],
-                      fileType:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.logsFileInfo?.["fileType"],
-                      expireDateTime:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.logsFileInfo?.["expireDateTime"],
-                      validationStatus:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.logsFileInfo?.["validationStatus"],
-                      validationFailureDetails:
-                        result.body.testArtifacts?.outputArtifacts
-                          ?.logsFileInfo?.["validationFailureDetails"],
-                    },
-              },
-        },
-    testResult: result.body["testResult"],
-    virtualUsers: result.body["virtualUsers"],
-    displayName: result.body["displayName"],
-    testId: result.body["testId"],
-    description: result.body["description"],
-    status: result.body["status"],
-    startDateTime: result.body["startDateTime"],
-    endDateTime: result.body["endDateTime"],
-    executedDateTime: result.body["executedDateTime"],
-    portalUrl: result.body["portalUrl"],
-    duration: result.body["duration"],
-    subnetId: result.body["subnetId"],
-    createdDateTime: result.body["createdDateTime"],
-    createdBy: result.body["createdBy"],
-    lastModifiedDateTime: result.body["lastModifiedDateTime"],
-    lastModifiedBy: result.body["lastModifiedBy"],
-  };
+  return result.body as any;
```

Comment:
This is temp workaround for PATCH LRO. Considering 1) there exists issues in getLroMetadata() for PATCH and 2) we don't have real case yet in DPG.

https://github.com/Azure/autorest.typescript/issues/2314